tests/unit/modules/test_event.py

```diff
@@ -0,0 +1,279 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import (
+    errors, response, utils,
+)
+from ansible_collections.sensu.sensu_go.plugins.modules import event
+
+from .common.utils import (
+    AnsibleExitJson, AnsibleFailJson, ModuleTestCase, set_module_args,
+)
+
+
+class TestGetObjects:
+    def testget_entity(self, mocker):
```

Comment:
Was this accidentally find-and-replaced when `_get_entity` became `get_entity`? ;) Same for both entity and both check tests :)

Reply:
heh whoops :blush: